packages/lit-dev-content/site/tutorials/content/tooltip/08.md

```diff
@@ -0,0 +1,74 @@
+The tooltip supports good performance via lazy creation, but using the
+`lazy` API is cumbersome. In addition, as implemented, the tooltip content
+doesn't update when the greeted person's name changes.
+
+A Lit directive can address these issues by encapsulating the tooltip's `lazy`
+API. Users of the directive will use an
+[element expression](http://lit.dev/docs/templates/expressions/#element-expressions)
+on the target element and provide a
+[TemplateResult](http://lit.dev/docs/libraries/standalone-templates/#rendering-lit-html-templates)
+containing the tooltip content.
+
+A scaffold for the directive is setup for you to fill in with the `update`
+method already completed. It stores directive `part` and the `tooltipContent`
+passed in so you can use them in the methods you'll add: `setupLazy` and
+`renderTooltipContent`.
+
+```ts
+update(part: ElementPart, [tooltipContent]: DirectiveParameters<this>) {
```

Comment:
Can probably remove this snippet since it's not actionable. You can probably also add comments in the `update` method. This step is already running longer than the others

Reply:
Done